src/ldc/eh/common.d

```diff
@@ -287,12 +287,72 @@ void _d_getLanguageSpecificTables(ubyte* data, ref ubyte* callsite, ref ubyte* a
     debug(EH_personality) printf("  - callsite: %p, action: %p, classinfo_table: %p, ciEncoding: %d\n", callsite, action, classinfo_table, ciEncoding);
 }
 
+// ---------------------------------------------------------------
+//    A fixed pool of T instances, used to optimise allocations
+//    Uses a free list to ensure fast de/allocation
+// ---------------------------------------------------------------
+template FixedPool(T, int N)
```

Comment:
Ah, right; there's no reason the interface has to be static. Good point. I'll find a space for the module + tests in `ldc.eh`.